server/src/main/java/io/druid/server/initialization/ZkPathsConfig.java

```diff
@@ -90,7 +90,7 @@ public String getConnectorPath()
     return (null == connectorPath) ?  defaultPath("connector") : connectorPath;
   }
 
-  protected String defaultPath(final String subPath)
+  public String defaultPath(final String subPath)
```

Comment:
accidentally included when I migrated PR stuff. will fix.

In general it is nice for extensions to be able to build a default ZK path using zkPathsConfig without extending zkpathsconfig itself, I will remove since it is out of scope of this pr